charts/stable/actualserver/values.yaml

```diff
@@ -2,12 +2,6 @@ image:
   repository: ghcr.io/actualbudget/actual-server
   pullPolicy: IfNotPresent
   tag: 24.2.0@sha256:e8c362dcc34498a5c44acd3184febe257d8e7f46781964b5705289195cba4d47
-securityContext:
-  container:
-    readOnlyRootFilesystem: false
-    runAsNonRoot: false
-    runAsGroup: 0
-    runAsUser: 0
```

Comment:
it likely needs readOnlyRootFilesystem: false
if it fails again, still requires root sadly.

Reply:
oh, interesting, I had checked the "ReadOnly File System" box in the install settings and it seemed to have worked fine on my machine. I'll re-add it then

Reply:
i think it needs root...https://github.com/truecharts/charts/actions/runs/7896237984/job/21550072709?pr=18237#step:10:205

Reply:
ah yes, I did have this file already on my system, so that's probably why it worked. I'll re-add the whole section then haha